File: Desktop/TecnPoint/TecnPoint.Modelos/DTO/ChamadoDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection;
using System.Runtime.InteropServices;
using System.Text;
using System.Threading.Tasks;
using TecnPoint.Modelos.Enum;

namespace TecnPoint.Modelos.DTO
{
    public class ChamadoDTO
    {
        public long idChamado { get; set; }
        public string descricao { get; set; }
        public string titulo { get; set; }
        public PrioridadeChamado prioridade { get; set; }
        public StatusChamado status { get; set; }
        public UsuarioDTO cliente { get; set; }
        public UsuarioDTO funcionario { get; set; }
        public Jornada jornada { get; set; }
        public Modulo modulo { get; set; }
    }
}

File: Backend/tecnpoint/src/main/java/com/techsolutions/tecnpoint/entities/DTO/ChamadoDTO.java
package com.techsolutions.tecnpoint.entities.DTO;

import com.techsolutions.tecnpoint.entities.Jornada;
import com.techsolutions.tecnpoint.entities.Modulo;
import com.techsolutions.tecnpoint.entities.enums.PrioridadeChamado;
import com.techsolutions.tecnpoint.entities.enums.StatusChamado;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class ChamadoDTO {

    Long idChamado;
    String descricao;
    String titulo;
    PrioridadeChamado prioridade;
    StatusChamado status;
    UsuarioDTO cliente;
    UsuarioDTO funcionario;
    Jornada jornada;
    Modulo modulo;
}
